src/routing.py

```diff
@@ -180,6 +184,48 @@ def _init_route_info(self):
         self.route_props.mod_formats_matrix = list()
         self.route_props.weights_list = list()
 
+    def load_k_shortest(self):
+        """
+        Load the k-shortest paths from an external file.
+        """
+        loaded_data = np.load('USB6014-10SP.npy', allow_pickle=True)
+        src_des = [int(self.sdn_props.source), int(self.sdn_props.destination)]
```

Comment:
I think by naming standards we should call this `src_des_list` due to it's data type